### PR TITLE
feat(content): Created two further one-liner spaceport missions

### PR DESCRIPTION
**Pull request by MasterOfGrey**
**Content (Missions)**

## Summary
These two conversations come to you courtesy of @TheMarksman-ES sending me some drafts to turn into a PR.

For concept criticism please direct your comments at him
For implementation including the mission structure, source locations or miscellaneous grammar fixes that'll be me.

### CODE REVIEW

File: data/human/culture conversations.txt [hunk 1]
```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
...
+	minor
+	source
+		government "Free Worlds"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The sun had already set when you walked out of the spaceport, trying to find a nearby restaurant to sample their wares. Looking around, the only source of light came from the building you walked out of, a beacon within the sea of darkness.`
+			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights before two men rush from an alley, pushing past you as they go. Before you could reply with a profanity, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, yet before you could do anything they had already disappeared, melting into the blackness beyond.`
```
Comment (roadrunner56) [inline data/human/culture conversations.txt:149]:
```suggestion
			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights. You are interrupted as two men rush from an alley, pushing past you as they go. Before you could respond, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, yet before you could do anything they had already disappeared, melting into the blackness beyond.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/human/culture conversations.txt [hunk 2]
```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
...
+	minor
+	source
+		government "Free Worlds"
+		attributes "frontier"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`As you land on the pad, you note a general lack of presence around the spaceport. No one scrambled to hook up your ship nor were there a full contingent waiting to receive you. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their overlords can find their yearly tribute.`
```
[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:133]: Yes, but if the pirates declare themselves to be their overlord when demanding tribute, then the note would identify them as their overlord, not as "a local pirate gang".

[the remark this answers is not in the document]
Comment (roadrunner56) [reply data/human/culture conversations.txt:133]: Yeah that's fair.

**Review by roadrunner56 — APPROVED** [review]

File: data/human/culture conversations.txt [hunk 3]
```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
...
+	minor
+	source
+		government "Free Worlds"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The sun had already set when you walked out of the spaceport, trying to find a nearby restaurant to sample their wares. Looking around, the only source of light came from the building you walked out of, a beacon within the sea of darkness.`
+			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights. You are interrupted as two men rush from an alley, pushing past you as they go. Before you could respond, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, yet before you could do anything they had already disappeared, melting into the blackness beyond.`
```
Comment (MCOfficer) [inline data/human/culture conversations.txt:149]:
```suggestion
			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights. You are interrupted as two men rush from an alley, pushing past you as they go. Before you can respond, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, yet before you could do anything they had already disappeared, melting into the blackness beyond.`
```

Slipped back into past tense here - also, is respond really appropriate? React, perhaps?

File: data/human/culture conversations.txt [hunk 4]
```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
...
+mission "Forest Fire"
+	minor
+	source
+		government "Free Worlds"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The sun had already set when you walked out of the spaceport, trying to find a nearby restaurant to sample their wares. Looking around, the only source of light came from the building you walked out of, a beacon within the sea of darkness.`
```
Comment (MCOfficer) [inline data/human/culture conversations.txt:148]: Not a fan of past tense, as usual.

**Review by MCOfficer — CHANGES_REQUESTED** [review]

File: data/human/culture conversations.txt (hunk 3 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:149]: I think responding is more active, and would be the relevant activity. You can pull a face and that's a reaction, but it doesn't achieve anything in the situation. Whereas a response would be something *useful* I feel.

File: data/human/culture conversations.txt (hunk 4 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:148]:
```suggestion
			`The sun has already set when you walk out of the spaceport, trying to find a nearby restaurant to sample their wares. Looking around, the only source of light comes from the building you just exited, a beacon within the sea of darkness.`
```

Comment (MCOfficer) [reply data/human/culture conversations.txt:148]: lgtm

File: data/human/culture conversations.txt (hunk 2 above)
Comment (Arrow2thekn33) [inline data/human/culture conversations.txt:133]:
```suggestion
			`As you land on the pad, you note a general lack of presence around the spaceport. No one scrambled to hook up your ship nor was there a full contingent waiting to receive you. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their overlords can find their yearly tribute.`
```

File: data/human/culture conversations.txt [hunk 5]
```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
...
+		random < 1
+	on offer
+		conversation
+			`As you land on the pad, you note a general lack of presence around the spaceport. No one scrambled to hook up your ship nor were there a full contingent waiting to receive you. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their overlords can find their yearly tribute.`
+			`	You decide against finding out where the inhabitants went.`
+				decline
+
+
+
+mission "Forest Fire"
```
Comment (Arrow2thekn33) [inline data/human/culture conversations.txt:139]: Why is this mission called Forest Fire?

**Review by Arrow2thekn33 — CHANGES_REQUESTED** [review]
Good missions, they really help characterize the free worlds.

File: data/human/culture conversations.txt (hunk 5 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:139]: Whoops

File: data/human/culture conversations.txt (hunk 2 above)
Comment (Anarchist2) [inline data/human/culture conversations.txt:133]: The mixed tense is kinda clunky.
```suggestion
			`As you land on the pad, you note that no one has scrambled to hook up your ship, nor has anybody else acknowledged your arrival. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their overlords can find their yearly tribute.`
```

File: data/human/culture conversations.txt [hunk 6]
```diff
@@ -120,6 +120,37 @@ mission "Paradise Cleaner Incident"
...
+	minor
+	source
+		government "Free Worlds"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The sun has already set when you walk out of the spaceport, trying to find a nearby restaurant to sample their wares. Looking around, the only source of light comes from the building you just exited, a beacon within the sea of darkness.`
+			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights. You are interrupted as two men rush from an alley, pushing past you as they go. Before you can respond, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, yet before you could do anything they had already disappeared, melting into the blackness beyond.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:149]:
```suggestion
			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights. You are interrupted as two men rush from an alley, pushing past you as they go. Before you can respond, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, long after all three people had already disappeared, melting into the blackness beyond.`
```

File: data/human/culture conversations.txt (hunk 2 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:133]: Can we re-try this one retaining the "general lack of presence"?

Comment (W1zrad) [reply data/human/culture conversations.txt:133]:
```suggestion
			`As you land on the pad, you note a general lack of presence around the spaceport. No one has scrambled to hook up your ship, nor has anybody else acknowledged your arrival. Instead, when you leave your ship, all you find is a note within the spaceport detailing where the "overlords" can find their yearly tribute.`
```
This look right? I also made a quick change at the bottom - it's not quite obvious who your `their` refers to.

File: data/human/culture conversations.txt (hunk 5 above)
Comment (W1zrad) [reply data/human/culture conversations.txt:139]: Still needs fixing

Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:139]: 💢

File: data/human/culture conversations.txt (hunk 2 above)
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:133]: I agree to the changes baring the "their," personally. It sounds right that it's "their" overlords over "the" overlords.
Then again, I'm not the one with Merge powers, so someone please give a second opinion.

File: data/human/culture conversations.txt (hunk 6 above)
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:149]: Looks right.

File: data/human/culture conversations.txt (hunk 5 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:139]:
```suggestion
mission "Missed Mugging"
```

File: data/human/culture conversations.txt (hunk 2 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:133]:
```suggestion
			`As you land on the pad, you note a general lack of presence around the spaceport. No one has scrambled to hook up your ship, nor has anybody else acknowledged your arrival. Instead, when you leave your ship, all you find is a note within the spaceport detailing where their "overlords" can find their yearly tribute.`
```

Comment (W1zrad) [reply data/human/culture conversations.txt]: Still not huge on the `their` vs `the`, but I'm cool with letting it slide

**Review by W1zrad — APPROVED** [review]
@MCOfficer all you mate

File: data/human/culture conversations.txt [hunk 7]
```diff
@@ -181,6 +197,21 @@ mission "Forest Fire"
...
+	minor
+	source
+		government "Free Worlds"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The sun has already set when you walk out of the spaceport, trying to find a nearby restaurant to sample their wares. Looking around, the only source of light comes from the building you just exited, a beacon within the sea of darkness.`
+			`	Choosing a random direction, you decide to walk down the quiet street, taking in the sights. You are interrupted as two men rush from an alley, pushing past you as they go. Before you can respond, you see a woman run out, clothes in disarray as she gives chase to the two men to no avail. Belatedly, you realize that they were holding a purse, long after all three people had already disappeared, melting into the blackness beyond.`
```
[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:210]: I'm partial to the first one.

**Review by MCOfficer — APPROVED** [review]

**Pull request merged**